.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: java
 jdk:
   - oraclejdk8
-after_success:
-  - bash .travis_after_success.sh
+#after_success:
+#  - bash .travis_after_success.sh
```

Comment:
Ah yes, but if you do it for this pull request, it as no chance of being accidentally merged and saves us time. We're lazy too!